tests/model_registry/rbac/conftest.py

```diff
@@ -159,18 +161,116 @@ def model_registry_group_with_user(
         yield group
 
 
+@pytest.fixture(scope="package")
+def user_credentials_rbac() -> dict[str, str]:
+    random_str = generate_random_name()
+    return {
+        "username": f"test-user-{random_str}",
+        "password": f"test-password-{random_str}",
+        "idp_name": f"test-htpasswd-idp-{random_str}",
+        "secret_name": f"test-htpasswd-secret-{random_str}",
+    }
```

Comment:
doesn't this force to have a single user in the idp for the whole execution, given the scope of the fixture?

Reply:
test_idp_user_session was a session scoped fixture originally. I changed it and any associated fixtures to package scoped. It can be changed to module too, since outside of test_mr_rbac.py, these fixtures are not used anywhere. This is creating the dict to be used with respect to test_idp_user_session. I don't understand the concern though, as it does not change any other users or logs in as this user.

Reply:
My concern is that this generates a single user for the idp. I'm confident that we will require more than a single user in the future, and this fixture (along with any related ones relying on this dict) will need to be updated